app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,43 @@
 class PostsController < ApplicationController
-  def index
+  def find_user
     @user = User.find(params[:user_id])
   end
```

Comment:
- [ ] Kindly use the `current_user` method created in the application controller.

Reply:
Hi @oloomoses ,

Thanks for this feedback.

Please I understand what you are saying, but I would like to explain something a bit. Imagine I am a user on Facebook and I create a post, should I be the one to comment on the post or other users should be the ones to comment on my post. That is the reason why I am using a random user to create the post, while the `current_user` is the one to comment or like the post. 

It would be nice if you considered this.

Thank you

Reply:
Hi @Gambit142, Sorry for the late reply. Here `current_user` means users who are currently logged in or users with active sessions.